apps/public-docsite-v9/src/Concepts/Upgrade/FromV0/Components/Input.stories.mdx

```diff
@@ -0,0 +1,81 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v0/Components/Input Upgrade" />
+
+# Input Upgrade
+
+Fluent UI Northstar (v0) provides the `Input` control to elicit input from users. Fluent UI v9 also provides an `Input` control but with a different API and feature set.
+
+## Examples
+
+### Basic Upgrade
+
+Basic usage of `Input` v0 looks like
+
+```tsx
+import * as React from 'react';
+import { Input, Text, Flex } from '@fluentui/react-northstar';
+
+const InputV0BasicExample = () => {
+  return <Input label="Search" />;
+};
+```
+
+An equivalent `Input` v9 usage is
+
+```tsx
+import * as React from 'react';
+import { Label, Input } from '@fluentui/react-components/unstable';
+import { useId } from '@fluentui/react-utilities';
+import { makeStyles } from '@griffel/react';
+
+const useLayoutStyles = makeStyles({
+  root: {
+    maxWidth: '300px',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+});
+
+const InputV9BasicExample = () => {
+  const layoutStyles = useLayoutStyles();
+  const inputId = useId('input');
+  return (
+    <div className={layoutStyles.root}>
+      <Label htmlFor={inputId}>Search</Label>
+      <Input id={inputId} />
+    </div>
+  );
+};
+```
+
+## Prop Mapping
+
+This table maps v0 `Input` props to the v9 `Input` equivalent.
+
+| v8                     | v9                                          | Notes                                                                                                       |
+| ---------------------- | ------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `accessibility`        | n/a                                         |                                                                                                             |
+| `as`                   | n/a                                         |                                                                                                             |
```

Comment:
The `as` prop should exist on the v9 `Input` AFAIK

Reply:
Technically the `as` prop exists on v9 but with the way all [Slot](https://github.com/microsoft/fluentui/blob/master/packages/react-utilities/src/compose/types.ts#L75-L113)s are defined on `Input` the element types cannot be changed. 

This makes sense to me since I don't think we'd ever want to render anything other than an `<input>` tag.

<img width="890" alt="input_as" src="https://user-images.githubusercontent.com/93940821/166485775-ecd0cf97-3a8d-4e89-92be-412617ef782b.png">